### ISSUE TITLE
Assess performance and memory prior to production release

### ISSUE THREAD

**Issue by ryanrdoherty** [VEuPathDB/service-sequence-retrieval#2]
Per @wbazant's comments in the original development PR, some items have not yet been addressed that need to be before this service goes into production.  Would be nice to have a new branch (from initial dev) for a developer to perform some testing and address concerns.  Wojtek's original notes:
```
Some missing work before this is production ready that I couldn't do:

  - running with a big index, seeing how much memory it needs (on disk they're 5x 200 MB at most so maybe 1 GB?)
  - checking that the requests for big queries properly stream the response back
  - checking what happens with a few parallel big requests - ideally they'd both process in parallel
  - observing peak memory usage - I think retrieving one while chromosome will cause the biggest jump per request, but actually, even metazoan chromosomes are just a few megabytes so it shouldn't be too bad
  - choosing good cutoff values for max num sequences and max num bases per request: I propose 1e6 and 1e8, but check it fits within 30 seconds
  - right-sizing the JVM setup and containerisation based on that
  - other devops (where this should run and what files it should have)
  - revisiting defline format in fasta: mostly, I think the deflines will come from request itself, so, .bed files should contain the desired header for each subsequence - the choices of "region only" or "query + region" are for user-defined bulk retrieval
```

**Comment by wbazant** [comment]
Related: submit a very long list of features, and make sure it's rejected without blowing up the container.

**Comment by aurreco-uga** [comment]
this has been released so closing
